### DISCUSSION TITLE
Gerätekommunikation gestört / Device communication disrupted

### DISCUSSION THREAD

**Discussion by timostr** [OpenCCU/OpenCCU#1955] [🇺🇸 Q&A]
### Describe the issue you are experiencing

After a  6 hours the connection to 2 HmIP-SAM sensors is lost. This is shown in service messages. All  my other 20 devices are working without any issues. Recently I've installed at a different Raspberry HomeAssistant and have integrated RaspberryMatic still runinng at RP3 using Homematic(IP) Local.
The sensors have new batteries and have been placed for test purpose close to the raspberry-Pi3.
[boot.log](https://github.com/jens-maus/RaspberryMatic/files/9638820/boot.log)
[messages.txt](https://github.com/jens-maus/RaspberryMatic/files/9638823/messages.txt)
[hmserver.txt.log](https://github.com/jens-maus/RaspberryMatic/files/9638824/hmserver.txt.log)

### Describe the behavior you expected

Device communication without errors.

### Steps to reproduce the issue

1.Take out batteries from HmIP-SAM
2.Put batteries back in
3.Wait for 6 hours
...

### What is the version this bug report is based on?

3.65.8.20220831

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Logs are attached above.
```

**Comment by albatrosify** [comment]
Same issue. But all of my actors show "Gerätekommunikation gestört".
Direkt-Verknüfpfung works.

**Comment by jens-maus** [comment]
Sorry, but this isn't an issue/bug in RaspberryMatic but rather a user problem which need to be discussed/analyzed in a discussion fora before concluding that this might be an issue in RaspberryMatic itself. In fact, I do have a HmIP-SAM myself and never had or have any issues with it. Thus, not a general issue, definitely!

**Reply by timostr** [reply]
Do you have a link to a guideline to analyse such problems further to find out the root cause? Is there anything in the provided logfiles to get a clue, why the communication is not working anymore after a couple of hours?
I have placed for testing the HMiP-SAM sensors 50 cm apart from  the RaspberryMatic device. 
What can I do as a user to correct this problem? Does it make sense to delete these sensors from RaspberryMatic, then reset the devices and register the devices again?
